packages/dappkit/src/index.ts

```diff
@@ -163,122 +159,3 @@ export async function requestTxSig(
 
   Linking.openURL(serializeDappKitRequestDeeplink(request))
 }
-
-function isValidPhoneNumber(phoneNumber: PhoneNumber): E164Number | undefined {
```

Comment:
Lookups are no longer feasible in dappkit given the quota-restricted salt queries required. I'm removing this until we figure out some alternative.